source-control/pull-request-templates/github/github-template.md

```diff
@@ -0,0 +1,27 @@
+# [Issue ID](./INSERT_URL_TO_ISSUE)
+
+<!-- Use the check list below to ensure your branch is ready for PR.  If the item is not applicable leave it put NA. -->
```

Comment:
This comment seems off.  

"If the item is not applicable leave it put NA."

Should it be

"If the item is not applicable leave it unchecked."

Reply:
good catch